apps/test/unit/gamelab/GameLabGroupTest.js

```diff
@@ -33,4 +34,237 @@ describe('GameLabGroup', function () {
       expect(sprite2.setSpeedAndDirection.calledWith(speed, direction)).to.be.true;
     });
   });
+
+  describe('collision methods', function () {
+    var sprite1, sprite2, targetSprite, group;
+
+    beforeEach(function () {
+      sprite1 = createSprite(0, 0, 100, 100);
+      sprite2 = createSprite(400, 400, 100, 100);
+      targetSprite = createSprite(200, 200, 100, 100);
+      group = createGroup();
+      group.add(sprite1);
+      group.add(sprite2);
+    });
+
+    describe('no sprite in group overlaps target sprite', function () {
+      it('returns false for isTouching', function () {
+        let result;
+        for (let i = 0; i < group.length + 1; i++) {
+          result = group.isTouching(targetSprite);
+        }
```

Comment:
When the function is called once (but has multiple sprites in the group), it will not return a value. It's only when `doneExec` is true (so when it's looped through all the sprites), that it will return the stored value on the state.
With the way we setup our JSInterpreter in tests, `doneExec` has no effect on recalling `isTouching` so it will only get called once (even though state gets stored). We call it n+1 times to mimic what happens in gamelab. I will add a comment to the code about this.